sdk/storage_blobs/src/blob/block_with_size_list.rs

```diff
@@ -126,4 +126,60 @@ mod test {
         assert!(bl.blocks[1].size_in_bytes == 62);
         assert!(bl.blocks[2].size_in_bytes == 62);
     }
+
+    #[test]
+    fn test_incomplete_response() {
```

Comment:
Can you add a comment here that explains that we want to explicitly test for responses that both have and don't have committed and uncommitted blocks? This is clear from this PR but might not be as clear in the future when someone is reading this code.

Reply:
updated